dynamo.py

```diff
@@ -721,3 +723,30 @@ def delete_emoji_role(emoji):
     )
     scan_for_roles()
     return "deleted"
+
+def addWelcomeMessage(user_id : int, message_id : int):
+    """Add a user's welcome message when they join
+
+    Parameters
+    ----------
```

Comment:
Can you create another util file (similar to dynamo.py) and call it something like cache.py? It would be helpful in the future when we add more features that require caching as opposed to permanent storage in dynamo

Reply:
Added it :)